Library/Homebrew/requirements/language_module_requirement.rb

```diff
@@ -12,11 +12,26 @@ def initialize(language, module_name, import_name = nil)
 
   satisfy(:build_env => false) { quiet_system(*the_test) }
 
-  def message; <<-EOS.undent
-    Unsatisfied dependency: #{@module_name}
-    Homebrew does not provide #{@language.to_s.capitalize} dependencies; install with:
-      #{command_line} #{@module_name}
+  def message
+    s = ""
+
+    s += <<-EOS.undent
+      Unsatisfied dependency: #{@module_name}
+      Homebrew does not provide special #{@language.to_s.capitalize} dependencies; install with:
+        #{command_line} #{@module_name}
     EOS
+
+    if @language == :rbx
+      s
+    elsif $?.exitstatus == 127
+      s += <<-EOS.undent
+        You may need to:
+          brew install #{@language}
+      EOS
+    s
+    else
+    s
+    end
```

Comment:
> How about:

:+1:. I didn't know if it would be cleaner/quicker for `rbx` users if we bailed out before checking the exit status, but the existing check is hardly slow so I can't imagine any performance reduction.

> string concatenation instead of +=.

We use the latter in a bunch of formulae and I try to keep the code between core and formulae relatively harmonious, but I'm fine with changing it.